contracts/libs/Owned.sol

```diff
@@ -13,5 +13,9 @@ contract Owned {
 
   function isOwner(address addr) public returns(bool) { return addr == owner; }
 
-  function transfer(address _owner) onlyOwner { owner = _owner; }
+  function transfer(address _owner) onlyOwner {
+    if (_owner != address(this)) {
```

Comment:
This was something that was recomended from the review. I agree that it's not likely, but it doesn't really hurt.